### PR TITLE
Add unit tests for ItemInfoUtils

### PR DESCRIPTION
**Pull request by iSpirydonava**
Added comprehensive unit tests for the ItemInfoUtils class, covering various scenarios and edge cases. The tests include methods for updating descriptions, finding attributes by resource, updating attributes, and checking attribute existence. This ensures the reliability and correctness of the ItemInfoUtils utility class.

### CODE REVIEW

File: src/test/java/com/epam/ta/reportportal/util/ItemInfoUtilsTest.java [hunk 1]
```diff
@@ -70,35 +69,110 @@ void shouldNotFindAttribute() {
...
     String key = "not-exist";
-    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(
-        getAttributeResources(), key);
+    Optional<ItemAttributeResource> itemAttributeResource = ItemInfoUtils.extractAttributeResource(getAttributeResources(), key);
     assertTrue(itemAttributeResource.isEmpty());
   }
 
+  @Test
+  void updateDescriptionTest() {
+    BulkInfoUpdateRQ.Description description = new BulkInfoUpdateRQ.Description();
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/util/ItemInfoUtilsTest.java:99]: Maybe in such cases, when we prepare data, make action and then make assertion, we can use "given-when-then" approach or "arrange-act-assert"

File: src/test/java/com/epam/ta/reportportal/util/ItemInfoUtilsTest.java [hunk 2]
```diff
@@ -70,35 +69,110 @@ void shouldNotFindAttribute() {
...
+    description.setAction(BulkInfoUpdateRQ.Action.UPDATE);
+    description.setComment("new comment");
+    Optional<String> updatedDescription = ItemInfoUtils.updateDescription(description, "existing description");
+    assertTrue(updatedDescription.isPresent());
```
Comment (pbortnik) [inline src/test/java/com/epam/ta/reportportal/util/ItemInfoUtilsTest.java:103]: Maybe we can add a message which will be displayed if assertion failed.
